endpoints/getting-started-grpc/helloworld/README.md

```diff
@@ -0,0 +1 @@
+[This code's documentation lives on the grpc.io site.](http://www.grpc.io/docs/quickstart/python.html)
```

Comment:
Why keep the code under `helloworld/`? Doesn't make sense with only one sample. Either way this README is not needed.